migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -23,17 +23,17 @@ Before you begin your migration, be sure to review the information on xref:../..
 .Prerequisites
 
 * The source cluster must be {product-title} 3.7, 3.9, 3.10, or 3.11.
-* The target cluster must be {product-title} 4.2.
+* The target cluster must be {product-title} {product-version}.
 * You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
 * You must have `podman` installed.
-* You must configure a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* The cluster on which the CAM tool is installed must have a 50 GB PVC for the MCG replication repository.
```

Comment:
We are looking more at confirming this.

@jmontleon please update as we learn what the default requirements are.